docs/tutorial/interaction_visualization_reporting/workdir/workflow/envs/download.yaml

```
@@ -0,0 +1,9 @@
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - python =3.11
+  - polars =1.1
+  - vega_datasets =0.9
+  - ipykernel =6.29
+  - notebook =7.2
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)